base_report_to_printer/views/ir_actions_report.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" ?>
+<odoo>
+    <record model="ir.ui.view" id="act_report_xml_view">
+        <field name="name">ir.actions.report.form (in base_report_to_printer)</field>
+        <field name="model">ir.actions.report</field>
+        <field name="inherit_id" ref="base.act_report_xml_view" />
+        <field name="arch" type="xml">
+            <page name="security" position="before">
+                <page string="Print" name="print">
+                    <group>
+                        <field name="property_printing_action_id" />
+                        <field name="printing_printer_id" />
+                        <field name="printer_tray_id" />
```

Comment:
Error when accessing to menu Settings > Technica > .. > Reports

![image](https://github.com/user-attachments/assets/d333cf8b-dfd0-42a8-8c0f-325b24f02718)